tools/add_triage_signature.py

```diff
@@ -839,6 +839,56 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class DownloadImage(ErrorSignature):
+    image_download_regex = re.compile(r"Host (.+?): New image status (.+?). result:.+?; download rate: (.+?) MBps")
+    minimum_download_rate_mb = 10
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="slow_image_download",
+            comment_identifying_string="h1. Host slow download rate:",
+        )
+
+    @classmethod
+    def _list_image_download_info(cls, events):
+        def get_image_download_info(event):
+            matches = cls.image_download_regex.findall(event["message"])
+            if len(matches) == 0:
+                return None
+            match = matches[0]
+            if len(match) < 3:
+                return None
```

Comment:
how can this be? I think it shouldn't return a match if it doesn't have all 3 groups